queries/authzed/highlights.scm

```diff
@@ -1,36 +1,53 @@
-((identifier) @keyword
- (#any-of? @keyword "definition" "permission" "relation"))
+((identifier) @function)
```

Comment:
```suggestion
(identifier) @function
```

That said, identifiers being functions and not variable as a baseline is definitely a new thing to me

Reply:
> identifiers being functions and not variable as a baseline [...]

Seems like a fair point, but I was kind of hoping not to add that to the list of differences between the file in the upstream repo and this repo.

Reply:
Ah, so it was a deliberate decision from the upstream authors. Not going to question that then.